theano/tensor/tests/test_basic.py

```diff
@@ -1683,7 +1718,7 @@ def test_py_c_match():
     expected_gamma = scipy.special.gamma
     expected_gammaln = scipy.special.gammaln
     expected_psi = scipy.special.psi
-    expected_chi2sf = lambda x, df: scipy.stats.chi2.sf(x, df).astype(x.dtype)
+    expected_chi2sf = lambda x, df: scipy.stats.chi2.sf(x, df)
```

Comment:
Hi @nouiz , I just revert it back. Is there anything that I need to change?
Thanks!